helpers/updateMemberRank.js

```diff
@@ -1,6 +1,7 @@
 // TODO: Pull these out to another location
 
 const { Configuration } = require('../services/configuration.js');
+const config2 = require('../config.json');
```

Comment:
I would rename this to something else than `config2`, maybe `cabbageQuantities` or something else that is a bit more descriptive?

If you plan to have other things than (awareded) cabbage quantities in this `config.json` file, maybe you can just append that (public) configuration to the `configuration.js` file and read it from the same `Configuration` property

Reply:
Renamed "Configuration" to "Environment", this way we can use the "Configuration" name to be used for the config.js file (afa170d)